src/core/Link/Link/Link.tsx

```diff
@@ -22,7 +22,7 @@ const StyledLink = styled(
     />
   ),
 )`
-  ${({ theme }) => baseStyles(theme)}
+  ${({ theme }) => LinkStyles(theme)}
```

Comment:
I actually find this naming quite informative compared to the bare `baseStyles` used throughout the library for all components. A more uniform naming would be to use baseStyles for now, but I could be discussed if we should change these names to be more informative in general.

Reply:
That is true. The current `baseStyles` gives thoughts that it is some base that should be appended with more styles into it.
I vote for for the naming changes all around the components to get this kind of more describing names. 👍

Reply:
Other components will be changed in later PR.